misc/python/materialize/checks/all_checks/__init__.py

```diff
@@ -10,11 +10,21 @@
 import glob
 from os.path import basename, dirname, isfile, join
 
+# file names with extension
+IGNORED_CHECK_FILE_NAMES = [
```

Comment:
As I wrote on Slack, I think it's better to derive the classes from `CheckDisabled`. Then it's explicit at the location of the check that it's disabled, not hidden in yet another location.